modules/appeals_api/config/schemas/v2/200995.json

```diff
@@ -155,7 +145,7 @@
               "attributes": {
                 "type": "object",
                 "properties": {
-                  "issue": { "allOf": [ { "$ref": "#/definitions/nonBlankString" }, { "maxLength": 140 } ] },
+                  "issue": { "$ref": "#/definitions/nonBlankString", "maxLength": 140 },
```

Comment:
I'm not sure this is a valid way to combine the requirements. Did you generate the docs and view it with the swagger ui? I feel like I tried something similar a bit ago and it wasn't happy about it 💥

Reply:
I believe combining the requirements like this is fine, but I could def be wrong. In this instance, however, the issues weren't generating so I copied and pasted what was in NOD since it works over in those parts!

Reply:
Hmm well since everything seems fine over in NOD land, I guess whatever explosions happened to me were for a different reason. Let's keep this as-is - it's nicer anyway 👍

Reply:
Thinking on it a bit more, maybe what I'm remembering was actually sending in more than the `maxLength` number of characters and it not catching it as invalid. I'll confirm.

Reply:
Ah, ok. I can look into it!

Reply:
Ok, at first glance it _does not_ appear to be catching it. I'll fix.